frontend/src/components/utils/DarkModeBtn.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+
+type DarkModeSwitchState = {
+  theme: string | null;
+};
+class DarkModeSwitch extends React.Component<unknown, DarkModeSwitchState> {
```

Comment:
To maintain consistency across all our components, it is better to convert that class component to a functional one